src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -286,9 +287,9 @@ public void CreateWalletWithWalletSetting100UnusedAddressBufferCreates100Address
         {
             DataFolder dataFolder = CreateDataFolder(this);
 
-            var walletManager = GetWalletManagerWithCustomConfigParam(dataFolder, "-walletaddressbuffer=100");
+            var walletManager = this.CreateWalletManager(dataFolder, Network.Main, "-walletaddressbuffer=100");
 
-            walletManager.CreateWallet("test", "mywallet", new Mnemonic(Wordlist.English, WordCount.Eighteen).ToString());
+            walletManager.CreateWallet("test", "mywallet", passphrase: new Mnemonic(Wordlist.English, WordCount.Eighteen).ToString());
```

Comment:
Here and in a few other places, the mnemonic is passed as the passphrase..

Reply:
I need to double check these as it happened accidentally when I did a resharper refactor

Reply:
It does suggest here that it would be better if the CreateWallet on the WalletManager accepted a Mnemonic type rather than a string for the mnemonic